librz/bin/bin.c

```diff
@@ -1293,3 +1293,149 @@ RZ_API const RzBinLdrPlugin *rz_bin_ldrplugin_get(RzBin *bin, const char *name)
 	}
 	return NULL;
 }
+
+static char *bin_demangle_cxx(RzBinFile *bf, const char *symbol, ut64 vaddr) {
```

Comment:
Please add `rz_return_val_if_fail(symbol)`

Reply:
Also why `bin_demangle_cxx()` is static while `bin_demangle_rust()` is `RZ_API`?

Reply:
mistake. it should be static.

Reply:
> Please add `rz_return_val_if_fail(symbol)`

This can't happen because the symbol value is checked by `rz_bin_demangle`